[2025-10-14T16:02:42.773+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: events_analytics_pipeline.dbt_docs_generate scheduled__2025-10-14T15:00:00+00:00 [queued]>
[2025-10-14T16:02:42.775+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: events_analytics_pipeline.dbt_docs_generate scheduled__2025-10-14T15:00:00+00:00 [queued]>
[2025-10-14T16:02:42.776+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2025-10-14T16:02:42.779+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): dbt_docs_generate> on 2025-10-14 15:00:00+00:00
[2025-10-14T16:02:42.782+0000] {standard_task_runner.py:60} INFO - Started process 2802 to run task
[2025-10-14T16:02:42.783+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'events_analytics_pipeline', 'dbt_docs_generate', 'scheduled__2025-10-14T15:00:00+00:00', '--job-id', '155', '--raw', '--subdir', 'DAGS_FOLDER/events_pipeline_dag.py', '--cfg-path', '/tmp/tmpgnwe564q']
[2025-10-14T16:02:42.785+0000] {standard_task_runner.py:88} INFO - Job 155: Subtask dbt_docs_generate
[2025-10-14T16:02:42.801+0000] {task_command.py:423} INFO - Running <TaskInstance: events_analytics_pipeline.dbt_docs_generate scheduled__2025-10-14T15:00:00+00:00 [running]> on host 7fd364930f15
[2025-10-14T16:02:42.826+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data-team' AIRFLOW_CTX_DAG_ID='events_analytics_pipeline' AIRFLOW_CTX_TASK_ID='dbt_docs_generate' AIRFLOW_CTX_EXECUTION_DATE='2025-10-14T15:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-14T15:00:00+00:00'
[2025-10-14T16:02:42.826+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-10-14T16:02:42.827+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /opt/***/event_gen && /home/***/.local/bin/dbt docs generate --profiles-dir /opt/***/.dbt --project-dir /opt/***/event_gen']
[2025-10-14T16:02:42.831+0000] {subprocess.py:86} INFO - Output:
[2025-10-14T16:02:43.399+0000] {subprocess.py:93} INFO - [0m16:02:43  Running with dbt=1.10.13
[2025-10-14T16:02:43.483+0000] {subprocess.py:93} INFO - [0m16:02:43  Registered adapter: clickhouse=1.9.4
[2025-10-14T16:02:43.611+0000] {subprocess.py:93} INFO - [0m16:02:43  Found 4 models, 30 data tests, 1 source, 599 macros
[2025-10-14T16:02:43.613+0000] {subprocess.py:93} INFO - [0m16:02:43
[2025-10-14T16:02:43.613+0000] {subprocess.py:93} INFO - [0m16:02:43  Concurrency: 4 threads (target='dev')
[2025-10-14T16:02:43.613+0000] {subprocess.py:93} INFO - [0m16:02:43
[2025-10-14T16:02:43.900+0000] {subprocess.py:93} INFO - [0m16:02:43  Building catalog
[2025-10-14T16:02:43.948+0000] {subprocess.py:93} INFO - [0m16:02:43  Catalog written to /opt/***/event_gen/target/catalog.json
[2025-10-14T16:02:44.980+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-10-14T16:02:44.994+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=events_analytics_pipeline, task_id=dbt_docs_generate, execution_date=20251014T150000, start_date=20251014T160242, end_date=20251014T160244
[2025-10-14T16:02:45.025+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-10-14T16:02:45.034+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
